wicket-core/src/test/js/all.html

```diff
@@ -181,8 +181,6 @@
 
 		<div id="usedAsContextWicket5025"></div>
 
-		<input type="text" id="inputChangeInput">
```

Comment:
why this is no more needed ? I think it is OK to keep this test for the `input` event

Reply:
Ignore this. It is better to remove some code.